### ISSUE TITLE
Width of columns

### ISSUE THREAD

**Issue by vodek3** [ronggang/transmission-web-control#271]
* Transmission: 2.94 (0), RPC: 15
* Tr Web Control version:  WEB Control: 1.6.0 beta(20180906)
* Browser name & version： Any
* Problem description: Width of columns

Is it possible to save the width of columns?
For example, as in the screenshot.
![width of columns](https://user-images.githubusercontent.com/25941601/46294678-f9211680-c59e-11e8-8500-0b98a8e1512a.jpg)

**Comment by Monter** [comment]
This may depend on the language chosen.
Too many default columns are active, in my opinion the defaults should be only basic and the remaining user could activate itself.

**Comment by disrupted** [comment]
yes, this is definitely needed! It would also be great if it could automatically fill the entire width that's available from the window
